src/components/project_admin/suite_settings.vue

```diff
@@ -16,120 +16,159 @@
   <div class="suite-settings">
     <div class="form-field-wrapper">
       <label class="label"> Suite name </label>
-      <validated-input ref="suite_name"
-                       v-model="d_suite.name"
-                       @input="$emit('field_change', d_suite)"
-                       :validators="[is_not_empty]">
+      <validated-input
```

Comment:
It looks like there are a lot of formatting-only changes to this file, which makes it harder for me to tell what actually changed in the file. Could you please revert the formatting changes?

Reply:
It looks like that specific formatting change was done in this [commit](https://github.com/eecs-autograder/ag-website-vue/pull/453/commits/860cf62c5081a92b3bd74a81bace0e22b449fa42) as part of a lot styling changes. Should I drop that commit via a rebase? Or should I just remove the new format changes for jsx and keep the rest in a new commit?

Reply:
Ah that makes sense--I had made a similar comment on Neil and Cay's other PR. If you're comfortable enough with the git wizardry to easily drop the commit, please do, otherwise I can deal with it when I make my next review pass.